docs/components/power-sensor/renogy.md

```diff
@@ -10,17 +10,15 @@ images: ["/icons/components/sensor.svg"]
 # SME: #team-bucket
 ---
 
-Configure a `renogy` sensor to integrate a [Renogy battery temperature sensor](https://www.amazon.com/Renogy-Battery-Temperature-Sensor-Controllers/dp/B07WMMJFWY) into your robot:
+Configure a `renogy` sensor to integrate a [Renogy battery temperature sensor](https://www.renogy.com/battery-temperature-sensor-for-renogy-solar-charge-controllers/) into your robot's system:
```

Comment:
```suggestion
Configure a `renogy` sensor to integrate a [Renogy battery temperature sensor](https://www.renogy.com/battery-temperature-sensor-for-renogy-solar-charge-controllers/) into your robot:
```

For this, I know "into your robot" is a little awkward. We're in the process of changing the language around this, so for now let's use the same language we use elsewhere (also makes the future find and replace easier).

Love the link change!